code/game/gamemodes/cult/ritual.dm

```diff
@@ -190,6 +190,7 @@ var/global/list/rnwords = list("ire","ego","nahlizet","certum","veri","jatkaa","
 	throw_range = 5
 	w_class = ITEMSIZE_SMALL
 	unique = 1
+	origin_tech = list(TECH_ARCANE = 6)

```

Comment:
This should, likely, be down at 2 maximum. It's a work of or pertaining to "Arcane" laws, though it's not really hyper anomalous in itself.